package.json

```diff
@@ -10,50 +10,53 @@
     "Alex Xia <khxia24@gmail.com>"
   ],
   "dependencies": {
-    "@material-ui/core": "^4.11.0",
-    "@material-ui/icons": "^4.9.1",
-    "@svgr/webpack": "^5.3.0",
-    "classnames": "^2.2.6",
-    "dayjs": "^1.10.4",
-    "gatsby": "^2.20.12",
-    "gatsby-image": "^2.3.1",
-    "gatsby-plugin-manifest": "^2.3.3",
-    "gatsby-plugin-material-ui": "^2.1.10",
-    "gatsby-plugin-react-helmet": "^3.2.1",
+    "@emotion/react": "^11.7.0",
+    "@material-ui/core": "^4.12.3",
+    "@material-ui/icons": "^4.11.2",
+    "@svgr/webpack": "^5.5.0",
+    "classnames": "^2.3.1",
+    "dayjs": "^1.10.7",
+    "false": "^0.0.4",
```

Comment:
What's this for?

Reply:
I honestly have no idea, not sure why this package was added, seems to be a mistake on my part